src/PaymentGateways/PayPalCommerce/PayPalClient.php

```diff
@@ -71,4 +70,18 @@ public function getApiUrl( $endpoint ) {
 
 		return "{$baseUrl}/$endpoint";
 	}
+
+	/**
+	 * Get PayPal homepage url.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @return string
+	 */
+	public function getHomePageUrl() {
+		return sprintf(
+			'https://www.%1$spaypal.com/',
```

Comment:
Let's remove the `www.` as it's not necessary.

Reply:
fixed